packages/erdpy/default.nix

```diff
@@ -0,0 +1,59 @@
+{pkgs}:
+with pkgs;
+  python3Packages.buildPythonApplication rec {
+    pname = "erdpy";
+    version = "2.0.3";
+
+    format = "wheel";
+    dist = "py3";
+    python = "py3";
+
+
+    src = python3Packages.fetchPypi {
+      inherit pname version format dist python;
+    
+      sha256 = "215edfb6f9f8c8214cc42e67e8d5328859486007ac4e3438cb9aabd21de67414";
+    };
+
+
+  postPatch = ''
```

Comment:
The indentation is a bit off here.

Reply:
Apologies about the indentation, nixpkgs-fmt wasn't working yesterday.

Reply:
Please switch to using `alejandra` for formatting code.